articles/routers.go

```diff
@@ -2,11 +2,12 @@ package articles
 
 import (
 	"errors"
-	"github.com/wangzitian0/golang-gin-starter-kit/common"
-	"github.com/wangzitian0/golang-gin-starter-kit/users"
-	"gopkg.in/gin-gonic/gin.v1"
 	"net/http"
 	"strconv"
+
+	"github.com/gin-gonic/gin"
```

Comment:
Why we need use the dev version? Version related issue I think `govendor` is better and enough, https://github.com/gothinkster/golang-gin-realworld-example-app/blob/master/vendor/vendor.json

Reply:
Now gin no longer gopkg style, use `govendor fetch github.com/gin-gonic/gin@v1.3` can get v1.3.x branch not develop version.

Reply:
Got, I have half year not write go...

Reply:
ohh, thanks a lot!